beetsplug/web/__init__.py

```diff
@@ -37,7 +37,10 @@ def _rep(obj, expand=False):
     out = dict(obj)
 
     if isinstance(obj, beets.library.Item):
-        del out['path']
+        if app.config.get('INCLUDE_PATHS', False):
+            out['path'] = util.displayable_path(out['path'])
+        else:
+            del out['path']
```

Comment:
Good catch on the encoding, I should have thought of that. Fixed now.

This project apparently does a really good job of Python 2/3 compatibility and encoding issues!